src/xrpld/app/misc/WasmVM.cpp

```diff
@@ -33,65 +33,70 @@
 
 namespace ripple {
 
+static void
+getReadOnlyWasmFunctions(HostFunctions* hfs, std::vector<WasmImportFunc>& i)
```

Comment:
No need to put it in the function as it was already in the function just dedicated for this purpose

Reply:
What do you mean?

I added this function to make it easier for future Smart Features/Smart Contracts to have a different collection of functions here. I grouped together all the ones that were almost certainly going to be included for any feature, and left the escrow-specific one separate.